frontend/src/components/custom-tools/header/Header.jsx

```diff
@@ -53,11 +53,24 @@ function Header({
   const { setPostHogCustomEvent } = usePostHogEvents();
 
   const [toolDetails, setToolDetails] = useState(null);
+  const [confirmModalVisible, setConfirmModalVisible] = useState(false);
+  const [lastExportParams, setLastExportParams] = useState(null);
 
-  const handleExport = (selectedUsers, toolDetail, isSharedWithEveryone) => {
+  const handleExport = (
+    selectedUsers,
+    toolDetail,
+    isSharedWithEveryone,
+    forcedExport = false
+  ) => {
+    setLastExportParams({
+      selectedUsers,
+      toolDetail,
+      isSharedWithEveryone,
+    });
```

Comment:
@jagadeeswaran-zipstack Can we set this inside the catch block, only when we want to show the force export modal?

Reply:
Yes. We can move it to catch block.